pkg/experiment/local/client.go

```diff
@@ -184,7 +184,7 @@ func (c *Client) FlagMetadata(flagKey string) map[string]interface{} {
 
 func (c *Client) doFlagsV2() (map[string]*evaluation.Flag, error) {
 	client := &http.Client{}
-	endpoint, err := url.Parse("https://api.lab.amplitude.com/")
+	endpoint, err := url.Parse(c.config.ServerUrl)
```

Comment:
Note: I'm unsure whether this route works with the EU server. If not, this is a breaking change.